content/en/logs/guide/getting-started-lwl.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Logging Without Limits™ Guide
+kind: guide
+disable_toc: true
+further_reading:
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "Learn more about Log Explorer"
+- link: "logs/explorer/patterns/"
+  tag: "Documentation"
+  text: "Learn about log patterns"
+- link: "logs/live_tail"
+  tag: "Documentation"
+  text: "Explore Live Tail"
+- link: "logs/logs_to_metrics"
+  tag: "Documentation"
+  text: "Learn how to generate metrics from ingested logs"
+---
+
+## Overview
+
+Datadog [Logging without Limits][1]™ provides flexibility by decoupling [log ingestion and indexing][2]. Cloud-based applications can generate logs at a rate of millions per minute. This type of high volume log indexing is helpful when monitoring every aspect of a service, but it can be inhibiting when troubleshooting during high priority issues or tracking bugs.
+
+This guide identifies key components of Logging Without Limits™ such as [Patterns](#2-identify-high-volume-logging-patterns), [Exclusion Filters](#3-create-a-log-pattern-exclusion-filter), [Custom log-based metrics](#4-generate-metrics-for-excluded-logs), and [Monitors](#create-an-anomaly-detection-monitor) that can help you better organize Log Explorer and monitoring your KPIs over time.
+
+## 1. Identify your most logging service status
+
+Your most logging service contains several logs, some of which may be irrelevant for troubleshooting. By identifying this service first, you can quickly track down which service status produces the most logs and is best to exclude from the [Log Explorer view][4].
+
+For example, you may want to investigate every 4xx and 5xx response code log, but excluded every 200 response code log from Log Explorer to expedite troubleshooting during a major outage or event.
+
+{{< img src="logs/guide/getting-started-lwl/identify_logging_service.gif" alt="Identify a most logging service status" responsive="true" style="width:100%;">}}
```

Comment:
This gif should be below the instructions in order to be consistent (in the layout) as other examples.